src/admin/menu/views/MenuOverview.tsx

```diff
@@ -1,115 +1,21 @@
 import { AdminLayout, AdminLayoutBody, AdminLayoutTopBarRight } from '../../shared/layouts';
-import { GET_MENU_OVERVIEW_TABLE_COLS, MENU_PATH } from '../menu.const';
-import { MenuOverviewTableCols } from '../menu.types';
+import { MENU_PATH } from '../menu.const';
 
 type MenuOverviewProps = DefaultSecureRouteProps;
 
 const MenuOverview: FunctionComponent<MenuOverviewProps> = ({ history }) => {
-	const { tText, tHtml } = useTranslation();
+	const { tText } = useTranslation();
 
-	const [menus, setMenus] = useState<any>([]);
-
-	const renderTableCell = (rowData: Partial<Avo.Menu.Menu>, columnId: MenuOverviewTableCols) => {
-		const placement = rowData.placement || undefined;
-
-		switch (columnId) {
-			case 'placement':
-				return (
-					<Link to={buildLink(MENU_PATH.MENU_DETAIL, { menu: placement })}>
-						{startCase(placement || '')}
-					</Link>
-				);
-			case 'actions':
-				return (
-					<ButtonToolbar>
-						<Button
-							icon="eye"
-							onClick={() =>
-								navigate(history, MENU_PATH.MENU_DETAIL, { menu: placement })
-							}
-							size="small"
-							title={tText(
-								'admin/menu/views/menu-overview___bekijk-de-navigatie-items-voor-deze-navigatie-balk'
-							)}
-							ariaLabel={tText(
-								'admin/menu/views/menu-overview___bekijk-de-navigatie-items-voor-deze-navigatie-balk'
-							)}
-							type="secondary"
-						/>
-						<Button
-							icon="plus"
-							onClick={() =>
-								navigate(history, MENU_PATH.MENU_ITEM_CREATE, { menu: placement })
-							}
-							size="small"
-							title={tText(
-								'admin/menu/views/menu-overview___voeg-een-navigatie-item-toe-aan-deze-navigatie-balk'
-							)}
-							ariaLabel={tText(
-								'admin/menu/views/menu-overview___voeg-een-navigatie-item-toe-aan-deze-navigatie-balk'
-							)}
-							type="secondary"
-						/>
-					</ButtonToolbar>
-				);
-			default:
-				return rowData[columnId];
-		}
-	};
-
-	const renderMenuOverview = (data: Partial<Avo.Menu.Menu>[]) => {
-		if (!data.length) {
-			return (
-				<ErrorView
-					message={tText(
-						'admin/menu/views/menu-overview___er-zijn-nog-geen-navigaties-aangemaakt'
-					)}
-				>
-					<p>
-						{tHtml(
-							'admin/menu/views/menu-overview___beschrijving-hoe-navigatie-items-toe-te-voegen'
-						)}
-					</p>
-					<Spacer margin="top">
-						<Button
-							icon="plus"
-							label={tText('admin/menu/views/menu-overview___navigatie-toevoegen')}
-							onClick={() => history.push(MENU_PATH.MENU_CREATE)}
-							type="primary"
-						/>
-					</Spacer>
-				</ErrorView>
-			);
-		}
-
-		setMenus(data);
-
-		return (
-			<Table
-				columns={GET_MENU_OVERVIEW_TABLE_COLS()}
-				data={data}
-				renderCell={(rowData: Partial<Avo.Menu.Menu>, columnId: string) =>
-					renderTableCell(rowData, columnId as MenuOverviewTableCols)
-				}
-				rowKey="id"
-				variant="bordered"
-			/>
-		);
-	};
+	const [menus] = useState<any>([]);
```

Comment:
Is this state still used? If it is, improve the typing.

Reply:
I think the top bar will need to be moved to the NavigationOverview component in admin-core-ui. So that the add button can be shown depending on the navigationBars that were fetched inside the NavigationOverview component
```
			{!!menus.length && (
				<AdminLayoutTopBarRight>
					<Button
						label={tText('admin/menu/views/menu-overview___navigatie-toevoegen')}
						onClick={() => history.push(MENU_PATH.MENU_CREATE)}
					/>
				</AdminLayoutTopBarRight>
			)}
```